csp-guru/datapackage.json

```diff
@@ -1,15 +1,167 @@
+                        "description": "solar collector assemblies (SCA) manufacturer"
+                    },
+                    {
+                        "name": "Financed_through",
+                        "description": "how the owner makes money from the generated electricity: PPA, FiT, spot sales"
+                    },
+                    {
+                        "name": "Remuneration",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], local currency"
+                    },
+                    {
+                        "name": "Remuneration_2015USD",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], 2015 USD "
+                    },
+                    {
+                        "name": "Remuneration_2015EUR",
+                        "description": "indicates the level of FiT or PPA in [ct/kWh], 2015 EURO "
+                    },
+                    {
+                        "name": "LCOE_5_2015USD",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 USD discounted"
+                    },
+                    {
+                        "name": "LCOE_10_2015USD",
+                        "description": "levelized cost of electricity with 10 years payback [ct/kWh], 2015 USD discounted"
+                    },
+                    {
+                        "name": "LCOE_5_2015EUR",
+                        "description": "levelized cost of electricity with 5 years payback [ct/kWh], 2015 EURO discounted"
+                    },
+                    {
+                        "name": "LCOE_10_2015EUR",
+                        "description": "levelized cost of electricity with 10 years payback [ct/kWh], 2015 EURO discounted"
+                    },
+                    {
+                        "name": "Other_support",
+                        "description": "financial assistence to project developer like tax credits, cheap credits, etc."
+                    },
+                    {
+                        "name": "Owner",
+                        "description": "owner of the CSP plant"
+                    },
+                    {
+                        "name": "Support_duration",
+                        "description": "duration of PPA, FiT, etc. [a]"
+                    },
+                    {
+                        "name": "Enipedia_ID",
+                        "description": "https://enipedia.org/wiki/Enipedia"
+                    },
+                    {
+                        "name": "SolarPACES_ID",
+                        "description": "SolarPACES_ID "
+                    },
+                    {
+                        "name": "Sources",
+                        "description": "indicates were the information for each plant was taken from"
+                    }
+                ]
+            }
         }
     ],
     "contributors": [
-        {
+      {
```

Comment:
This change is wrong.

Reply:
Hey, yes, I don't even know what's that.